validate_yamls.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python
+
+import argparse
+import glob
+import os
+import sys
+
+from lib import exception
+from lib.utils import is_package_installed
+from lib.utils import run_command
+
+def validate_package_metadata(package_metadata_file_path):
+
+    try:
+        run_command("yamllint %s" % package_metadata_file_path)
+    except exception.SubprocessError as e:
+        #pylint: disable=no-member
+        print("validation of package metadata file %s failed, output: %s" % (package_metadata_file_path, e.stdout))
+        return
+
+def validate_packages_metadata(base_dir):
```

Comment:
As the package metadata yaml file format is ours, we could use function this to validate our syntax too. Anyway, as were are not doing this for now, I agree with your suggestion.